bulk-user-setup/bulk-user-creation.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+CSV_FILE_PATH="$1"
+ADMIN_USER="$2"
+ADMIN_USER_PWD="$3"
+IDAM_CLIENT_SECRET="$4"
+ENV=${5:-prod}
+
+REDIRECT_URI="https://create-bulk-user-test/oauth2redirect"
+CLIENT_ID="bulk-user-test"
+
+function usage() {
+  echo "usage: ./bulk-user-creation.sh <user-csv-file-path> <ccd-admin-user> <ccd-admin-user-pwd> <idam-client-secret> <optional-env>"
+  echo "example: ./bulk-user-creation.sh /Users/xyz/bulk-users.csv ccd-test-admin@mail.com test-password bulk-users-client-secret aat"
+}
+
+function get_idam_url() {
+    if [ "$ENV" == "prod" ]
+    then
+      url="https://idam-api.platform.hmcts.net"
+    else if [ "$ENV" == "local" ]
+    then
+      url="http://localhost:5000"
+    else
+      url="https://idam-api.${ENV}.platform.hmcts.net"
```

Comment:
based on usage: ENV is optional: then if we come here with out passing ENV value then this URL might be wrong..